src/develop/imageop.h

```diff
@@ -114,10 +114,10 @@ typedef enum dt_iop_flags_t
 /** status of a module*/
 typedef enum dt_iop_module_state_t
 {
-  dt_iop_state_HIDDEN = 0, // keep first
-  dt_iop_state_ACTIVE,
-  dt_iop_state_FAVORITE,
-  dt_iop_state_LAST
+  DT_IOP_STATE_HIDDEN = 0, // keep first
```

Comment:
As we are at it, we can remove `DT_` from here for consistency.

Reply:
alrighty, will do